OpenChange/iCalEvent+MAPIStore.m

```diff
@@ -137,6 +137,107 @@ - (void) _setupEventAlarmFromProperties: (NSDictionary *) properties
     }
 }
 
+- (int) _updateFromAttendeeMAPIProperties: (NSArray *) recipients
+                                 withRole: (NSString *) role
+                                 outParam: (BOOL *) organizerIsSet
+{
+  NSDictionary *dict;
+  NSString *attEmail;
+  iCalPerson *person;
+  iCalPersonPartStat newPartStat;
+  NSNumber *flags, *trackStatus;
+  int i, effective;
+
+  effective = 0;
+  for (i = 0; i < [recipients count]; i++)
```

Comment:
I understand your point but that code is not mine (has always been there), it's just moved. I think that kind of changes are out of the scope of this fix. In encourage you to create a new branch called for example ejhernandez/type-conversion-sanity to fix as many of this situations as you want if you are really concerned about this. By the way, we are not losing any precision on that comparison, there is no problem on doing a comparison like char < unsigned long, neither gcc or clang will warn even if -Wconversion is passed.